core/services/job/models.go

```diff
@@ -458,3 +464,37 @@ type BlockhashStoreSpec struct {
 	// UpdatedAt is the time this job was last updated.
 	UpdatedAt time.Time `toml:"-"`
 }
+
+// BootstrapSpec defines the spec to handles the node communication setup process.
+type BootstrapSpec struct {
+	ID                                     int32              `toml:"-"`
+	ContractID                             string             `toml:"contractID"`
+	Relay                                  relaytypes.Network `toml:"relay"`
+	RelayConfig                            RelayConfig
+	MonitoringEndpoint                     null.String     `toml:"monitoringEndpoint"`
+	BlockchainTimeout                      models.Interval `toml:"blockchainTimeout"`
+	ContractConfigTrackerSubscribeInterval models.Interval `toml:"contractConfigTrackerSubscribeInterval"`
```

Comment:
there is no subscribe interval in ocr2, we should actually remove it in the generic ocr2 spec as well